backend/src/main/scala/cromwell/backend/io/WorkflowPaths.scala

```diff
@@ -3,15 +3,15 @@ package cromwell.backend.io
 import java.nio.file.{Path, Paths}
 
 import com.typesafe.config.Config
-import cromwell.backend.{BackendJobDescriptorKey, BackendWorkflowDescriptor}
+import cromwell.backend.BackendWorkflowDescriptor
 
 object WorkflowPaths{
   val DockerRoot = Paths.get("/root")
 }
 
 class WorkflowPaths(workflowDescriptor: BackendWorkflowDescriptor, config: Config) {
   // TODO will need update when configuration is rearranged
```

Comment:
Not sure myself... pinging @gauravs90 for comment on whether it should be saved or not. I'll defer to deleting the line unless we get clarification on keeping it.